packages/devtools/src/commands/executeScript.js

```diff
@@ -0,0 +1,37 @@
+import command from '../scripts/executeScript'
+import { transformExecuteArgs, transformExecuteResult } from '../utils'
+import { SERIALIZE_PROPERTY, SERIALIZE_FLAG } from '../constants'
+
+export default async function executeScript ({ script, args }) {
+    const page = this.windows.get(this.currentWindowHandle)
+    const scriptTimeout = this.timeouts.get('script')
+
+    const executePromise = page.$eval(
+        'html',
+        command,
+        script,
+        SERIALIZE_PROPERTY,
+        SERIALIZE_FLAG,
+        ...transformExecuteArgs.call(this, args)
+    )
+
+    const timeoutPromise = new Promise((_, reject) => {
+        setTimeout(
+            () => {
+                const additionalTimeoutInfo = (
```

Comment:
this is `false` or a `string`, I think we should use a ternary here so it will either return the said `string`or a empty `string`

Reply:
then we can also simplify the code below it:
```
return reject(new Error(`script timeout ${additionalTimeoutInfo}`))
```